### PR TITLE
Update Master branch to Pastas 1.3.0

### PR DESCRIPTION
**Pull request by raoulcollenteur**
# Short Description

This PR updates Pastas master - branch to 1.3.

# Checklist before PR can be merged:
- [x] is documented
- [x] Format code with [Black formatting](https://black.readthedocs.io)
- [x] type hints for functions and methods
- [x] tests added / passed
- [x] Example Notebook (for new features)
- [x] Remove output for all notebooks with changes
- [x] ReadTheDoc looks good?

### CODE REVIEW

File: pastas/io/base.py [hunk 1]
```diff
@@ -118,11 +118,22 @@ def _load_model(data: dict) -> Model:
         n = getattr(ps.noisemodels, data["noisemodel"].pop("class"))()
         ml.add_noisemodel(n)
 
-    # Add fit object to the model
+    # Add solver object to the model from pas-files < 1.3.0  TODO Deprecate
```
Comment (dbrakenhoff) [inline pastas/io/base.py:121]: Is this a future ToDo or a 1.3 ToDo? Perhaps indicate which version number this todo is meant for?

File: pastas/version.py [hunk 2]
```diff
@@ -4,13 +4,13 @@
 
 logger = logging.getLogger(__name__)
 
-__version__ = "1.2.0"
+__version__ = "1.3.0b"
```
Comment (dbrakenhoff) [inline pastas/version.py:7]: 1.3.0

**Review by dbrakenhoff — APPROVED** [review]

File: pastas/version.py (hunk 2 above)
Comment (raoulcollenteur) [reply pastas/version.py]: Yes! Just changed it :)

File: pastas/io/base.py (hunk 1 above)
Comment (raoulcollenteur) [reply pastas/io/base.py:121]: That is one to do in 1.4! Such that people can use 1.3 to save to the new format.. So all good.

**Comment by dbrakenhoff** [comment]
We might need this in plotting.__init__?

```python
"""The plotting sub-package contains plot methods and tools for Pastas.

.. currentmodule:: pastas.plotting

.. autosummary::
    :toctree: ./generated
    :nosignatures:

    plots
    modelplots
    modelcompare
    plotutil
    plotly

"""
```

**Pull request merged**